### ISSUE TITLE
Unable to get the plugin to work.

### ISSUE THREAD

**Issue by linkpy** [pyrochlore/obsidian-tracker#30]
We are trying to use Tracker to make some graph related to helpdesk agent performances.

We have a collection of notes, each one of them is a review of an incident created by an agent, and has the following frontmatter :

```yaml
agent: a_name
incident: a_string
sysid: another_string
incident creation date: 2021-06-21T06:13:56
```
Note: the note's name is the incident string (thus not a date).

After looking for a while on the doc, we were unable to get that graph to work :
```yaml
searchType: frontmatter, frontmatter
searchTarget: incident creation date, agent
datasetName: xDataset, agentDataset
folder: Analysis/Incidents
bar:
    title: "Bar Chart"
    xAxisLabel: Date
    yAxisLabel: Count
```

Over and over again we have "No notes found under the given search condition".
They are notes in the given folder. Each of them have the right frontmatter (they are generated by a script) so we don't know what we are doing wrong.
Could you help us?

**Comment by pyrochlore** [comment]
Hi @linkpy, 
Is it the number of occurrences of 'agent: a_name' in front matter you want to count?
Currently tracker only collects 'values' from a front matter key.

But you can use searchType `text` to count 'agent: a_name'.
For example,

``` tracker
searchType: frontmatter, text
searchTarget: 'incident creation date, agent: a_name'
dateFormat: YYYY-MM-DD HH:mm:ss
datasetName: xDataset, agentDataset
xDataset: 0
folder: case/linkpy/data
bar:
    title: "Bar Chart"
    xAxisLabel: Date
    yAxisLabel: Count
```

You can also use output type `summary` to get data like total count.
 
Also note that, the plugin does not accept ISO 8601 date format for now. 
So to make the example above works, we have to remove the character 'T' in the 'incident creation date' fields in notes.

**Comment by pyrochlore** [comment]
Will support date format in ISO 8601

**Comment by pyrochlore** [comment]
ISO-8601 date format was supported in the new released version 1.8.0.
[Here](https://github.com/pyrochlore/obsidian-tracker/blob/master/examples/TestDateFormats.md#iso-8601-date-format) is an example of it.
Enjoy tracking.

**Comment by pyrochlore** [comment]
Will close this issue.
If problems remain unresolved, welcome to reopen the issue.
